### PR TITLE
Regex for dead detection

### PR DESCRIPTION
**Pull request by Akababa**
Partially addresses #26

### CODE REVIEW

**Comment by wanderrful** [comment]
I support this change because now I can fart myself to death!

File: story/utils.py [hunk 1]
```diff
@@ -41,23 +41,11 @@ def get_num_options(num):
...
-
-    dead_phrases = ["you die", "You die", "you died", "you are dead", "You died", "You are dead", "You're dead",
-                    "you're dead", "you have died", "You have died", "you bleed out"]
-    for phrase in dead_phrases:
-        if phrase in text:
-            return True
-    return False
+    text = text.lower()
+    you_dead_regexps = ["you('re| are) (dead|killed)", "you die", "you('ve| have) (died|been killed)",
+                        "you \w+ to death"]
```
Comment (wanderrful) [inline story/utils.py]: I support this change because now I can fart myself to death!

Comment (ben-bay) [reply story/utils.py]: Hehe

Add `"you \w+ yourself to death"`

Comment (Akababa) [reply story/utils.py]: Sure why not xD

**Comment by ben-bay** [comment]
Soft reset onto current master and recommit to make history a bit cleaner.

```
git remote add upstream git://github.com/nickwalton/AIDungeon.git
git fetch upstream
git reset upstream/master
git add -A
git commit
git push -f
```

**Comment by Akababa** [comment]
Honestly I don't know what happened, but it's a one-line change so if you want you can just copy and paste it.

**Pull request closed without merging**